R/pkg/R/DataFrame.R

```diff
@@ -2685,7 +2686,8 @@ setMethod("unionAll",
 
 #' Union two or more SparkDataFrames
 #'
-#' Union two or more SparkDataFrames. This is equivalent to \code{UNION ALL} in SQL.
+#' Union two or more SparkDataFrames by row. In constrast to \link{union}, this method
+#' requires that the input SparkDataFrames have the same column names.
```

Comment:
I'd just say, `as in R's rbind, this method requires...`
btw, should we care about data type matching - does R's rbind check?